src/util/githubFetcher.ts

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
 // To avoid throwing errors with 404.
-axios.defaults.validateStatus = (_) => true;
+axios.defaults.validateStatus = function () { return true };
```

Comment:
@helio-frota I do think you can just write `axios.defaults.validateStatus = () => true;`

I'm not a big fun of the `= function () { ... }` syntax cause it feels wired

If you have a function expression just using an arrow function I think it's the best approach

Reply:
if I'm not wrong that is causing lint issues also.. but I'll look again

Reply:
thank you @alexalikiotis , suggestion added, code updated :+1: